app/assets/javascripts/discourse-loader.js

```diff
@@ -146,9 +146,6 @@ var define, requirejs;
       "@ember/object/internals": {
         guidFor: Ember.guidFor,
       },
-      "@ember/test-helpers": {
```

Comment:
Don't think this change should be part of this PR

Reply:
It should be - it's defined now in `test-shims` including the visit helper. It doesn't need to be in the global loader anymore.